Project/Front-End/career-connect/src/components/JobCard.jsx

```diff
@@ -1,15 +1,15 @@
 
 
 
-export default function JobCard() {
+export default function JobCard({jobtitle, company, location, description}) {
 
     return (
       <><div className="job-card">
-        <h1>Job Title</h1>
-        <h2>Company</h2>
-        <h3>Location</h3>
+        <h2>{jobtitle}</h2> 
```

Comment:
This may need some conditional rendering to ensure the card does not display unless the props being rendered on the screen are actually provided.